File: src/main/java/com/matchFit/user/entity/SignUpRequest.java
package com.matchFit.user.entity;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class SignUpRequest {
    private String email;
    private String password;
    private String nickname;
    private Integer age;
    private String gender;
    private String town;
    private String sports;
    private boolean isKakaoUser;
}

File: src/main/kotlin/com/matchFit/user/dto/request/SignUpRequest.kt
package com.matchFit.user.dto.request


class SignUpRequest {
    lateinit var email: String
    var password: String? = null
    lateinit var nickname: String
    var age: Int = 0
    lateinit var gender: String
    lateinit var town: String
    lateinit var sports: String
    var isKakaoUser: Boolean = false
}
